README.md

```diff
@@ -53,85 +71,70 @@ The Optimism Immunefi program offers up to $2,000,042 for in-scope critical vuln
 │   └── <a href="./packages/sdk">sdk</a>: provides a set of tools for interacting with Optimism
+│   └── <a href="./packages/web3js-plugin">web3js-plugin</a>: Adds functions to estimate L1 and L2 gas
+├── <a href="./patches">patches</a>: Is this used somewhere or should it be deleted?
 ├── <a href="./proxyd">proxyd</a>: Configurable RPC request router and proxy
 └── <a href="./specs">specs</a>: Specs of the rollup starting at the Bedrock upgrade
+└── <a href="./ufm-test-services">ufm-test-services</a>: Runs a set of tasks to generate metrics
 </pre>
 
-## Branching Model
+## Development and Release Process
 
-### Active Branches
+### Overview
 
-| Branch          | Status                                                                           |
-| --------------- | -------------------------------------------------------------------------------- |
-| [master](https://github.com/ethereum-optimism/optimism/tree/master/)                   | Accepts PRs from `develop` when intending to deploy to production.                  |
-| [develop](https://github.com/ethereum-optimism/optimism/tree/develop/)                 | Accepts PRs that are compatible with `master` OR from `release/X.X.X` branches.                    |
-| release/X.X.X                                                                          | Accepts PRs for all changes, particularly those not backwards compatible with `develop` and `master`. |
+Please read this section if you're planning to fork this repository, or make frequent PRs into this repository.
 
-### Overview
+### Production Releases
+
+Production releases are always tags, versioned as `<component-name>/v<semver>`.
+For example, an `op-node` release might be versioned as `op-node/v1.1.2`, and an L1 smart contract release might be versioned as `contracts-L1/v1.0.0`.
+Release candidates are versioned in the format `op-node/v1.1.2-rc.1`.
+We always start with `rc.1` rather than `rc`.
+
+Tags of the form `v<semver>`, such as `v1.1.4`, indicate releases of all Go code only, and **DO NOT** include smart contracts.
+This naming scheme is required by Golang.
+In the above list, this means these `v<semver` releases contain all `op-*` components, and exclude all `contracts-*` components.
 
-This repository generally follows [this Git branching model](https://nvie.com/posts/a-successful-git-branching-model/).
-Please read the linked post if you're planning to make frequent PRs into this repository.
+`op-geth` embeds upsteam geth’s version inside it’s own version as follows: `vMAJOR.GETH_MAJOR GETH_MINOR GETH_PATCH.PATCH`.
+Basically, geth’s version is our minor version.
+For example if geth is at `v1.12.0`, the corresponding op-geth version would be `v1.101200.0`.
+*Note that we pad out to three characters for the geth minor version and two characters for the geth patch version.
+Since we cannot left-pad with zeroes, the geth major version is not padded.
 
-### Production branch
+The full set of components that have releases are:
```

Comment:
We also have `ci-builder`, `proxyd`, and maybe more.
Take a look at the options in https://github.com/ethereum-optimism/optimism/actions/workflows/tag-service.yml

Reply:
Thanks, I've added all the missing packages and sorted them alphabetically in 1b564fbff

Reply:
Also pushed d19fb00f7 to link to the node operator docs